config/core/test-configs.yaml

```diff
@@ -1825,6 +1831,14 @@ test_configs:
       - sleep_mem
       - smc
 
+  - device_type: asus-cx9400-volteer
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - cros-ec
+      - kselftest-lib
```

Comment:
OK https://staging.kernelci.org/test/plan/id/62126a8ed8a6809a194b313a/

Reply:
There's a problem with the clang build though:
https://staging.kernelci.org/test/plan/id/62126f8c86aa5f3d764b3157/

Reply:
It worked with stable 5.10 though so this looks like a kernel problem in mainline:
https://lava.collabora.co.uk/scheduler/job/5717633